### DISCUSSION TITLE
AttributeError: 'NoneType' object has no attribute 'rsplit' when using AwsLambdaInstrumentor().instrument()

### DISCUSSION THREAD

**Discussion by ganesh-kumolus** [open-telemetry/opentelemetry-python-contrib#2769] [General]
When initiation
  AwsLambdaInstrumentor().instrument()
Got issue 
 AttributeError: 'NoneType' object has no attribute 'rsplit'

**Comment by emdneto** [comment]
If you're getting this because of the package `opentelemetry-instrumentation-aws-lambda` being installed by default, we provided a partial fix to the issue here https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2750. But if you are really using the AwsLambdaInstrumentor, you probably should run the code using this https://github.com/open-telemetry/opentelemetry-lambda/blob/main/python/src/otel/otel_sdk/otel-instrument, which will set the environment variables you need.
